src/Compilers/VisualBasic/Portable/Declarations/DeclarationTable.vb

```diff
@@ -279,6 +279,19 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+        Public Shared Function ContainsName(
+            mergedRoot As MergedNamespaceDeclaration,
+            name As String,
+            filter As SymbolFilter,
+            cancellationToken As CancellationToken) As Boolean
+
+            ' note: because we want to do case-insensitive lookup, and because our decl tables
+            ' do not store members in a way that can lookup in a case-insensitive manner, we have
+            ' to defer to just calling the normal predicate version.
+            Return ContainsName(
+                mergedRoot, Function(n) IdentifierComparison.Equals(n, name), filter, cancellationToken)
```

Comment:
Note: https://github.com/dotnet/roslyn/pull/26330 is an approach that address this.